Packs/CommonPlaybooks/ReleaseNotes/2_2_15.md

```diff
@@ -0,0 +1,14 @@
+
+#### Playbooks
+##### New: Account Enrichment - Generic v2.1
+Enrich accounts using one or more integrations.
```

Comment:
Let's change the RN to something else, we don't need the whole playbook description

Reply:
Removed the word `new`